frontend/new-cm-sg-ls-vm/cm_setup_0.1.sh

```diff
@@ -7,13 +7,17 @@
 yum -y update
 
 #Install and setup the Sumo Collector
-wget https://teradeploy.blob.core.windows.net/binaries/SC_1.0.zip -P /tmp/
-unzip -o /tmp/SC_1.0.zip
-rpm -i sumo/SumoCollector-19.182-25.x86_64.rpm
-cp sumo/user.properties /opt/SumoCollector/config/
-cp sumo/sumo_cm_vm.json /opt/SumoCollector/config/
+mkdir /tmp/sumo
+wget "https://collectors.sumologic.com/rest/download/linux/64" -O /tmp/sumo/SumoCollector.sh && sudo chmod +x /tmp/sumo/SumoCollector.sh
+wget "$3/user.properties" -O /tmp/sumo/user.properties
```

Comment:
Could you add a comment at the top of the file describing what $2 and $3 are expected to be?